doc/source/conf.py

```diff
@@ -173,6 +173,7 @@ def find_test_modules(package_path):
     "writing-documentation": "contributor-how-to-write-documentation.html",
     "apiref-binaries": "ref-api-cli.html",
     "fedbn-example-pytorch-from-centralized-to-federated": "example-fedbn-pytorch-from-centralized-to-federated.html",
+    "how-to-use-built-in-middleware-layers": "how-to-use-built-in-mods.html",
```

Comment:
I see `middleware-layer` here

Reply:
yes, this is an intentional redirect in case anyone used the old url